data/json/professions.json

```
@@ -1121,7 +1121,7 @@
     "type": "profession",
     "ident": "k9_cop",
     "name": "K9 Officer",
-    "description": "You spent your career busting drug smugglers with your faithful canine companion.  Now the world has ended and none of that matters anymore.  But at least you have a loyal friend.",
+    "description": "You spent your career busting drug smugglers with your faithful canine companion.  Now the world has ended, and none of that matters anymore.  Your loyal dog is still with you, though, ready to face the Cataclysm together.",
```

Together feels a little awkward to me.
```suggestion
    "description": "You spent your career busting drug smugglers with your faithful canine companion.  Now the world has ended, and none of that matters anymore.  Your loyal dog is still with you, though, ready to face the Cataclysm with you.",
```